examples/quickstart/human_feedback.ipynb

```diff
@@ -31,7 +31,9 @@
     "from trulens.apps.custom import TruCustomApp\n",
     "from trulens.core import TruSession\n",
     "\n",
-    "session = TruSession()"
+    "session = TruSession()\n",
+    "session.reset_database()\n",
+    "session.start_dashboard(force=True, port=8080)"
```

Comment:
please revert the reset_database and start_dashboard args.

Reply:
done